platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/properties/PropertiesLoader.java

```diff
@@ -118,28 +118,45 @@ public Properties loadProperties(String propertiesFile) {
   }
 
   /**
-   * Load properties from a file using the given classloader. The System Property placeholders will
-   * also be replaced by their actual values.
+   * Will attempt to load properties from a file using the given classloader. If that fails, several
+   * other methods will be tried until the properties file is located.
    *
    * @param propertiesFile the resource name or the file path of the properties file
    * @param classLoader the class loader with access to the properties file
    * @return Properties deserialized from the specified file, or empty if the load failed
    */
   public Properties loadProperties(String propertiesFile, ClassLoader classLoader) {
-    return loadProperties(propertiesFile, classLoader, true);
+    Properties properties = new Properties();
+    if (propertiesFile == null) {
+      LOGGER.debug("Properties file must not be null.");
+      return properties;
+    }
+
+    Iterator<BiFunction<String, ClassLoader, Properties>> strategiesIterator =
+        PROPERTY_LOADING_STRATEGIES.iterator();
+    do {
+      properties = strategiesIterator.next().apply(propertiesFile, classLoader);
+    } while (properties.isEmpty() && strategiesIterator.hasNext());
+
+    properties = substituteSystemPropertyPlaceholders(properties);
+    return properties;
   }
 
   /**
-   * Will attempt to load properties from a file using the given classloader. If that fails, several
-   * other methods will be tried until the properties file is located.
+   * Will attempt to load properties from a file using the given classloader without replacing the
+   * system properties with their value. If that fails, several other methods will be tried until
+   * the properties file is located.
+   *
+   * <p>Note that loading the properties directly from the file system will replace the system
+   * properties with their corresponding values so if that method is used to load the properties,
+   * the system properties will be substituted.
```

Comment:
❓ I'm confused. Isn't this method only ever going to be used when loading the properties from the file system? So won't this always happen when using this method?